.github/workflows/master_build.yml

```diff
@@ -0,0 +1,114 @@
+name: Master CI Build
+
+on:
+  push:
+    branches:
+#      - master
```

Comment:
you'd have to uncomment this to make it work on the master branch

Reply:
This is one of the things that need to be changed before actually merging.
I am just leaving all the "old" settings in for now, so weblate can still be used on this branch.